.github/workflows/lapis2.yml

```diff
@@ -57,7 +57,9 @@ jobs:
         uses: docker/metadata-action@v4
         with:
           images: ${{ env.DOCKER_IMAGE_NAME }}
-          tags: type=ref,event=branch
+          tags: |
+            type=ref,event=branch
+            type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', 'main') }}
```

Comment:
Why `format('refs/heads/{0}', 'main')` instead of just `refs/heads/main`?

Reply:
Good question, I changed it.